server/server.go

```diff
@@ -337,7 +339,9 @@ func Server(db *sql.DB) *martini.ClassicMartini {
 	}
 	// proxy to logtail
 	if os.Getenv("LOGTAIL_SERVICE_HOST") != "" && os.Getenv("LOGTAIL_SERVICE_PORT") != "" {
-		m.Post("/tails/:app", ProxyToLogTail)
+		m.Post("/tails", ProxyToLogTail)
+		m.Post("/tails/:id", ProxyToLogTail)
+		m.Get("/tails/:id", ProxyToLogTail)
```

Comment:
Yes, that handler just proxies back to a specific host, so we mount which paths on the region api we want to go back to which host and use the function PorxyToLogTail to specificy the host we'd like to go to.  It will automatically pickup the path from the passed in URL and preserve it while passing it on.